### PR TITLE
API-79 Document `/authorize` parameters

### PR DESCRIPTION
**Pull request by bertramakers**
### Added

- Added an overview of all the possible URL parameters for `/authorize`

---

Ticket: https://jira.uitdatabank.be/browse/API-79

### CODE REVIEW

**Comment by bertramakers** [comment]
@brusselsregular maybe good if you also take a look at this sometime

File: projects/authentication/docs/user-access-token.md [hunk 1]
```diff
@@ -271,6 +279,39 @@ To learn more about the Authorization Code Flow with PKCE, see the [the Auth0 do
...
+| `redirect_uri` | For example `https://your-application.com/authorize` | **Required** | A URL of your application starting with `https://` to redirect the user back to after logging in. A `code` query parameter with the authorization code that can be exchanged for a user access token by your application will be appended at the end.
+| `code_challenge` | String generated from a cryptographically-random code verifier. | Required if using **PKCE** | See the [PKCE example](#example-1) for more info.
+| `code_challenge_method` | `S256` | Required if using **PKCE** | **Must** be set to `S256`. See the [PKCE example](#example-1) for more info.
+| `prompt` | `none` (default) or `login` | Optional | If set to `login`, the user will always be asked to login even if they still have an active session on the authorization server. Useful to force the user to login again, which also makes it possible to switch to another account as before.
+| `referrer` | `museumpas`, `uitdatabank` or `uit` | Optional | A publiq brand, which is used to set the background image of the login page accordingly.
```
Comment (corneelwille) [inline projects/authentication/docs/user-access-token.md:299]: `uitpas` is probably also a possible value?

**Review by corneelwille — APPROVED** [review]
Looks good! Note that I can only review these docs from an "external" perspective since I'm not familiar with all the ins and outs of the `/authorize` endpoint.

File: projects/authentication/docs/user-access-token.md (hunk 1 above)
Comment (bertramakers) [reply projects/authentication/docs/user-access-token.md:299]: Ah yes, and `uitdatabank` should actually be `udb` now that I look at it again in the Confluence page where this was documented before. And there is also `cultuurkuur`. I'll fix it!

Comment (bertramakers) [reply projects/authentication/docs/user-access-token.md:299]: I see why I made these mistakes, I accidentally documented the values for the same `referrer` parameter on the old custom `/login-from-app` that is documented on the same Confluence page

**Comment by bertramakers** [comment]
> Looks good! Note that I can only review these docs from an "external" perspective since I'm not familiar with all the ins and outs of the `/authorize` endpoint.

I also didn't know about most of these parameters, I took them from https://confluence.uitdatabank.be/pages/viewpage.action?pageId=26197640 which @brusselsregular shared with me in the ticket

File: projects/authentication/docs/user-access-token.md (hunk 1 above)
Comment (bertramakers) [reply projects/authentication/docs/user-access-token.md]: Fixed now!

**Comment by bertramakers** [comment]
> Note that I can only review these docs from an "external" perspective since I'm not familiar with all the ins and outs

This is also good by the way, because then you can see if it is understandable if you don't have experience with it

**Pull request merged**